src/Uno.UI.Tests/Windows_UI_Xaml/Given_ResourceDictionary.cs

```diff
@@ -684,5 +685,13 @@ public void When_XamlControlsResources()
 			Assert.IsTrue(xcr.ContainsKey(typeof(Button)));
 			Assert.IsInstanceOfType(xcr[typeof(Button)], typeof(Style));
 		}
+
+		[TestMethod]
+		public void When_Needs_Eager_Materialization()
+		{
+			Assert.IsFalse(TestInitializer.IsInitialized);
+			var control = new Test_Control_With_Initializer();
```

Comment:
Codacy found an issue: [Remove the unused local variable 'control'.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=6073998)